pkg/frr/frr.go

```diff
@@ -514,12 +510,17 @@ func tearDownVrf(vrf *infradb.Vrf) bool {
 		log.Printf("FRR Deleted event")
 		delCmd1 := fmt.Sprintf("no router bgp %+v vrf %s", localas, path.Base(vrf.Name))
 		delCmd2 := fmt.Sprintf("no vrf %s", path.Base(vrf.Name))
-		_, err = Frr.FrrBgpCmd(ctx, fmt.Sprintf("configure terminal\n %s\n exit\n", delCmd1))
-		if err != nil {
+		data, err = Frr.FrrBgpCmd(ctx, fmt.Sprintf("configure terminal\n %s\n exit\n", delCmd1))
+		if strings.Contains(data, "Can't find BGP instance") { // Trying to delete non exist VRF return true
+			return true
+		}
+		if err != nil || checkFrrResult(data, false) {
```

Comment:
if we need to check frr for each frr cmd, why not move that check into FrrBgpCmd/FrrZebraCmd and other frr functions?

as well as `strings.Contains(`?

Reply:
Done